src/yival/data_generators/openai_prompt_data_generator.py

```diff
@@ -199,31 +211,40 @@ def generate_examples(self) -> Iterator[List[InputData]]:
                     )
                 for r in responses:
                     self.process_output(
-                        r["choices"][0]["message"]["content"], all_data, chunk
+                        r["choices"][0]["message"]["content"],
+                        all_data,  # pyright: ignore
+                        chunk,
+                        self.config.fixed_input  # pyright: ignore
                     )
-            else:
-                with tqdm(
-                    total=self.config.number_of_examples,
-                    desc="Generating Examples",
-                    unit="example"
-                ) as pbar:
-                    # call_option = self.config.call_option if self.config.call_option else {}
-                    output = llm_completion(
-                        Request(
-                            model_name=self.config.model_name,
-                            prompt=messages,
-                            params=self.config.call_option  #type:ignore
-                        )
-                    ).output
-                    self.process_output(
-                        output.choices[0].message.content, all_data, chunk
+        else:
```

Comment:
I have tested and found that pbar in "Generating Examples" works well, while pbar in "Generating Variations" cannot auto-update. I can fix it in the next pr.